src/native-common/FrontLayerViewManager.tsx

```diff
@@ -169,13 +169,14 @@ export class FrontLayerViewManager {
                                 bottom: y + height, width: width, height: height };
 
                         // Find out if the press event was on the anchor so we can notify the caller about it.
-                        if (touchEvent.pageX >= anchorRect.left && touchEvent.pageX < anchorRect.right
-                            && touchEvent.pageY >= anchorRect.top && touchEvent.pageY < anchorRect.bottom) {
+                        if (touchEvent.pageX && touchEvent.pageY &&
```

Comment:
These checks will not do the right thing if the values happen to be defined and equal to zero. I think you need to do explicit isDefined checks here.